lib/WebLedgerClient.js

```diff
@@ -3,6 +3,7 @@
  */
 'use strict';
 
+const _get = require('lodash.get');
```

Comment:
Is there any chance we can do without lodash?

Reply:
I know you know the technical answer to this question is 'of course', but why do you ask?  I thought I was doing pretty well by not including the full lodash package.

Reply:
I will add that `_.get` is a familiar to the DB tech stack.  It's a very useful API.

Reply:
Hehe I appreciate just the get and not the full package :)

My main question is - why even do get? How is it different from the `||` operator? In this case, `this.ledgerAgent.service, serviceId` ?

Reply:
Right, good question.  I didn't want to get into that detail, but we do have `plugins` that get nested into the `service` object.  so when you're looking for a serviceId, it might be something like `plugin.somePlugin.foo`, in which case you would plugin that string literal here, and it will do the right thing.

Reply:
The routes I'm talking about are setup here: https://github.com/digitalbazaar/bedrock-ledger-agent/blob/master/lib/http.js#L526-L547

Reply:
Can you say more? Having trouble picturing this (re string literal)

Reply:
Perhaps looking at the `_.get` API docs would help: https://lodash.com/docs/4.17.11#get

Reply:
I did, yeah. I'm still not seeing the benefit over `||`.